docs/topics/blocklist.md

```diff
@@ -22,9 +23,11 @@ with v3 blocklist this is a bloomfilter that is queried - if the addon guid and
 v2/v1 are referred to as "legacy" blocklist in these docs.
 ```
 
-AMO holds the addon blocklist records and generates the bloomfilters as needed, which are then uploaded to Remote Settings. The records are managed via the admin tools on addons-server.
+AMO holds the addon blocklist records (`Block` models) defining blocks for the **v3 system** (including hard/soft type) and generates the v3 bloomfilters and stashes as needed, which are then uploaded to the `addons-bloomfilters` collection on Remote Settings. The `Block` records are managed via the admin tools on addons-server.
```

Comment:
> (`Block` models) 
did you mean `Block` records?  Or `Block` model records?

Reply:
The records in question are instances of the Block model.

Reply:
There a Block instances (Block model instances), or Block records; there's is only one Block model (i.e. not models).

Reply:
Updated.